pkg/common/secret/utils.go

```diff
@@ -0,0 +1,51 @@
+package secret
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GenerateGenericSecret(secretName string, secretNamespace string,
+	key string, value string) (*corev1.Secret, error) {
```

Comment:
If you're not running the create, why do you need an error?

Reply:
You're right. Removed it